var/spack/repos/builtin/packages/py-merlin-info/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyMerlinInfo(PythonPackage):
+    """A custom version of Phillip J. Eby's setuptools."""
+
+    homepage = "https://pypi.org/project/merlin_info/"
+    url      = "https://pypi.io/packages/source/m/merlin_info/merlin_info-1.8.tar.gz"
```

Comment:
This URL doesn't work, use:
```python
import os

url = "file://{0}/merlin-1.8.tar.gz".format(os.getcwd())
```
This will allow people who already have the tarball to install it using a local copy. You should also add a note saying that the package used to be available on PyPI but the name "merlin" was reappropriated for a different software package.